src/MOIWrapper.jl

```diff
@@ -0,0 +1,357 @@
+export ECOSOptimizer
+
+using MathOptInterface
+const MOI = MathOptInterface
+const CI = MOI.ConstraintIndex
+const VI = MOI.VariableIndex
+
+const MOIU = MOI.Utilities
+
+const SF = Union{MOI.SingleVariable, MOI.ScalarAffineFunction{Float64}, MOI.VectorOfVariables, MOI.VectorAffineFunction{Float64}}
+const SS = Union{MOI.EqualTo{Float64}, MOI.GreaterThan{Float64}, MOI.LessThan{Float64}, MOI.Zeros, MOI.Nonnegatives, MOI.Nonpositives, MOI.SecondOrderCone, MOI.ExponentialCone}
+
+struct Solution
+    ret_val::Int
+    primal::Vector{Float64}
+    dual_eq::Vector{Float64}
+    dual_ineq::Vector{Float64}
+    slack::Vector{Float64}
+    objval::Float64
+end
+Solution() = Solution(0, Float64[], Float64[], Float64[], Float64[], NaN)
+
+mutable struct Data
```

Comment:
Same comments as with SCS.jl about the name of the struct and commenting on its purpose.